### ISSUE TITLE
Assert That Logical Error with registerOptions

### ISSUE THREAD

**Issue by DanSimonet** [JasperHG90/qualtRics#48]
Hey Jasper. I am very eager and excited to try out the qualtrics API integration package. However, I am stuck on step 1. I run the following code with my token and institution url:

registerOptions(api_token = "my-api-token",
                          root_url = "montclair.co1.qualtrics.com")

But it throws the error:
Error: assert_that: assertion must return a logical value

I have tried with config file and changing the url, but I continue receiving this error. I feel like I'm missing something obvious.

**Comment by JasperHG90** [comment]
Hi Dan,

Thanks for your message.

That error occurs because of a mistake in my code somewhere. Could you try to install the GitHub version `r devtools::install_github("JasperHG90/qualtRics")` and try again?

Also, it'd be helpful to know what your .yml file looks like.

Thanks,

J.

**Comment by DanSimonet** [comment]
Thanks Jasper.

Following my post last night I tried connecting with the `qualtricsR` and `qsurvey` packages. I was successful connecting with `qualtricsR` but encountered a version error installing the development version of `qsurvey`. It looked something similar to this.
```
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  namespace 'httpuv' 1.3.3 is being loaded, but >= 1.3.5 is required
```
I updated R to version 3.4.1 (2017-06-30) and was able to successfully install and use `qsurvey`. 

With the updated R version, I re-installed the GitHub version of `qualtRics`, ran the code, and it worked like a charm! Am able to directly pull Qualtrics data. I deleted the .yml last night as `qualtRics` won't load if it encounters an error reading the config file. However, I used a text file with the following set-up:
```
api_token: 99999 (fake api code)
root_url: montclair.co1.qualtrics.com
verbose: TRUE
uselabels: TRUE
convertstandardcolumns: TRUE
uselocaltime: FALSE
datewarning: TRUE
```
